tests/test_files/test_not_unique.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+def test_not_uniq() -> None:
```

Comment:
No need for type annotation in test files.

Reply:
Fixed